src/libraries/System.Runtime.Numerics/src/System/Numerics/BigInteger.cs

```diff
@@ -2461,6 +2482,54 @@ private static bool GetPartsForBitManipulation(ref BigInteger x, ref Span<uint>
             return x._sign < 0;
         }
 
+        private static bool IsEmpty([NotNullWhen(false)] uint[]? bits)
```

Comment:
Should it be `IsNull` instead? Maybe we can inline the `bits == null` check at the callsite (as there is only one caller).

Reply:
Yeap, you are right. I'll fix it after complete review